ansible/roles/installation/coronabar/files/coronabar.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
Its better to use ansible modules then scripts and shell commands:
https://www.ansible.com/blog/ansible-best-practices-essentials

you can just use ansible snap modyle instead of having a shell script:
https://docs.ansible.com/ansible/latest/collections/community/general/snap_module.html#snap-module
https://stackoverflow.com/questions/47305658/install-snap-packages-with-ansible